src/app/Shared/Services/Api.service.tsx

```diff
@@ -808,3 +820,5 @@ export interface MatchedCredential {
   matchExpression: string;
   targets: Target[];
 }
+
+export const UPLOADS_SUBDIRECTORY: string = 'uploads';
```

Comment:
> The WebServer now points to the beta RecordingGetHandler and ReportGetHandler endpoints when the WebServer#getArchivedDownloadUrl and WebServer#getArchivedReportUrl methods are called, respectively. This means archived recording-related notifications will now contain target-specific download URLs. Notifications have also been updated so that the target field is equal to "uploads" rather than the empty string for uploaded recordings. This is because for the new target-specific handlers the sourceTarget parameter cannot be set to the empty string in place of a connectUrl for uploaded recordings. Since the "uploads" string is the logical alternative to use for this parameter, in order to maintain consistency, notifications also explicitly identify a recording as having originated as an upload directly into Cryostat's archives.

I am thinking we could summarize Hareet's comment above and add it here for reference.